src/Core/Domain/Attachment/Query/GetAttachmentInformationList.php

```diff
@@ -32,6 +32,8 @@
 
 /**
  * Query providing attachments information
+ *
+ * @todo: this possibly should be deprecated. It is not used and product attachments usecase changed, it shouldn't be needed anymore.
```

Comment:
todo. Can i deprecate this? :up:

Reply:
It's not used anywhere and I can't even see some related behat test So yes I would deprecate this one and remove it in next major to avoid keeping dead code